apps/zotonic_mod_authentication/priv/dispatch/dispatch

```diff
@@ -19,6 +19,9 @@
  {logon_reminder,   ["logon", "reminder"],                controller_template, [ {template, "logon.tpl"}, seo_noindex, is_logon ]},
  {logon_confirm,    ["logon", "confirm"],                 controller_template, [ {template, "logon.tpl"}, seo_noindex, is_logon ]},
 
+ % Simple session overview
+ {sessions, ["sessions"], controller_template, [ {template, "sessions.tpl"}, seo_noindex, {acl, is_auth} ]},
```

Comment:
In the 0.x it was:

```erlang
 {sessions_user, ["logon", "sessions"], ...}]}
```

Maybe we can give it the same name and path on the 1.x?

Reply:
6d46918d36cf9d602b9b9efbe64c1e78781e78b8 👍